_layouts/contact_us.html

```diff
@@ -18,12 +18,21 @@
   </div>
 {% endif %}
 
+{% if site.contact_unplanned_outage %}
+  <div class="usa-alert usa-alert--warning">
+    <div class="usa-alert__body">
+      <p class="usa-alert__text">{{ page.unplanned_outage_content}}</p>
+    </div>
+  </div>
+{% endif %}
+
 {{ page.help_center_content | markdownify }}
 
 <contact-us-form
   maintenance-alert-id="contact-us-maintenance-alert"
   maintenance-start-time="{{ site.contact_maintenance_start_time }}"
   maintenance-end-time="{{ site.contact_maintenance_end_time }}"
+  {% if site.contact_unplanned_outage %} unplanned-outage hidden {% endif %}
```

Comment:
Oh I like doing this entirely in markup! 👍 

Two additional thoughts:

- We don't appear to need the `unplanned-outage` attribute anymore
- Maybe we avoid rendering this entire element altogether if there's an outage?

i.e.

```html
{% unless site.contact_unplanned_outage %}
  <contact-us-form></contact-us-form>
{% endunless %}
```

Reply:
Yeah I like your solution better because it will make it a lot easier for new folks to see what is happening.